DataProcession.py

```diff
@@ -0,0 +1,68 @@
+import re
+from ShowResult import ShowReult
+
+class DataProcessing:
+    """This class calculate the different reports as per format"""
+    def max_min_calculate(self, data):
+        """Calculate the highest, lowest temprature resp. and most humidity 
+
+        Args:
+            data ([list]): [required data on which we do processing ]
+
+        Returns:
+            [Results]: [result storage class]
+        """
+        self.index1 = data.names.index('Max TemperatureC')
```

Comment:
index1 name is not self explanatory. Please update these variable names to some meaningful variable